test/e2e/scheduling/predicates.go

```diff
@@ -368,8 +368,18 @@ var _ = SIGDescribe("SchedulerPredicates [Serial]", func() {
 
 	It("validates that a pod with an invalid NodeAffinity is rejected", func() {
 		By("Trying to launch a pod with an invalid Affinity data.")
+		validationChangedVersion := utilversion.MustParseSemantic("v1.10.2")
```

Comment:
Apologies, I might have misunderstood as well. Why can't we check the reverse - like skip this test if version is greater than 1.9 for example?

Reply:
Thanks for the feedback. Since I'm working on the release-1.9 branch, I don't think there's much value in adding new utils. I would not want to cherry-pick it forward, so I would end up writing the util in master, picking it backward, then fixing this test.